charts/spire/charts/spire-server/README.md

```diff
@@ -22,7 +22,7 @@ A Helm chart to install the SPIRE server.
 | autoscaling.maxReplicas | int | `100` |  |
 | autoscaling.minReplicas | int | `1` |  |
 | autoscaling.targetCPUUtilizationPercentage | int | `80` |  |
-| bundleConfigMap | string | `"spire-server"` |  |
+| bundleConfigMap | string | `"spire-bundle"` |  |
```

Comment:
The default was wrong here but overridden in the main values. Once removed from the main values, showed up as broken here.

Reply:
good catch